lib/chromedriver.js

```diff
@@ -343,12 +352,14 @@ class Chromedriver extends events.EventEmitter {
         }
         // this is a chrome above the latest version we know about,
         // and we have a chromedriver that is beyond what we know,
-        // so use the most recent chromedriver that we found
-        const {version, executable} = cds[0];
-        log.warn(`No known Chromedriver available to automate Chrome version '${chromeVersion}'.\n` +
-          `Using Chromedriver version '${version}', which has not been tested with Appium` +
-          autodownloadMsg);
-        return executable;
+        // so usee the most recent chromedriver that we found
+        if (!_.isEmpty(cds) && !cds[0].minChromeVersion) {
+          const {version, executable} = cds[0];
+          log.warn(`No known Chromedriver available to automate Chrome version '${chromeVersion}'.\n` +
```

Comment:
Seems like this would be better as two `log.warn` calls, rather than one with a newline in it (the newlines are just split anyway).

Reply:
yep, easy